accounts/api.py

```
@@ -36,47 +92,395 @@ def get(self, request: Request, *args, **kwargs):
+                        summary="Simple example",
+                        description="Example response for lists",
+                        value=PAGINATED_LIST_EXAMPLE,
+                        response_only=True,
+                    ),
+                ],
+            ),
+            404: OpenApiResponse(description="Account not found"),
+            500: OpenApiResponse(description="Internal server error"),
+        },
+    )
+    def get(self, request: Request, *args, **kwargs):
```

Ruff (0.14.5)

Unused method argument: `args`

(ARG002)